skimage/data/__init__.py

```diff
@@ -319,6 +320,20 @@ def hubble_deep_field():
     return load("hubble_deep_field.jpg")
 
 
+def retina():
```

Comment:
We had a quick discussion with @ThomasWalter to change the name. It's actually conflicting with #2496 where I found useful to add a full retina image, something that can be used in many other examples as well.

I suggest to change the name of the function + the png file, and keep the rest as it is.

Reply:
This has been changed to `microaneurysms` (function and image name).